docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -240,26 +240,27 @@ That's all! Your hosted cluster is now created.
 Check the status of your hosted cluster via:
 
     ```bash
-    $ oc get hostedclusters -n local-cluster
+    oc get hostedclusters -n local-cluster
     ```
 
 ## Importing the Hosted cluster into MCE
 
-Hosted clusters are automatically imported into MCE once the control plane becomes available. All ACM addons are also enabled if ACM is installed. 
+Hosted clusters are automatically imported into MCE once the control plane becomes available. All ACM addons are also enabled if ACM is installed.
 
-#### Disabling automatic import
+### Disabling automatic import
 
 1. On the hub cluster, edit the `AddonDeploymentConfig` resource  `hypershift-addon-deploy-config` in the `hypershift` namespace.
+
    ```bash
-    $ oc edit addondeploymentconfig hypershift-addon-deploy-config -n multicluster-engine
+    oc edit addondeploymentconfig hypershift-addon-deploy-config -n multicluster-engine
```

Comment:
Nitpick, the indentation for the commands aren't all the same
![image](https://github.com/stolostron/hypershift-addon-operator/assets/15898988/21d173d9-1e43-42e8-9b0a-37ea0cb4f00f)